source/shared/net/references/developer-tools.rst

```diff
@@ -0,0 +1,35 @@
+.. _developer-tools:
+
+===============
+Auxiliary Tools
+===============
+
+.. contents::
+   :local:
+   :backlinks: none
+
+
+Decrypt encrypted output
+========================
+
+Some Concordium tools, such as the Mobile Wallet, sometimes produce encrypted output. These files can be decrypted and inspected using the **utils** tool . You can download the tool from :ref:`Installation downloads <downloads>`.
+
+To encrypt, enter:
+
+``utils encrypt --in input.example --out output.json``
```

Comment:
I think my previous comment was misleading. To be consistent with the decrypt line this should be
```
utils encrypt --in input.json --out output.example
```